src/components/UserSelector.tsx

```diff
@@ -20,21 +81,19 @@
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
         <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">
-            Leanne Graham
-          </a>
-          <a href="#user-2" className="dropdown-item is-active">
-            Ervin Howell
-          </a>
-          <a href="#user-3" className="dropdown-item">
-            Clementine Bauch
-          </a>
-          <a href="#user-4" className="dropdown-item">
-            Patricia Lebsack
-          </a>
-          <a href="#user-5" className="dropdown-item">
-            Chelsey Dietrich
-          </a>
+          {!error &&
+            users.map(user => (
+              <a
+                key={user.id}
```

Comment:
Avoid generating keys on render as it's mentioned in the checklist. Here, you are using `user.id` as a key which is fine as long as `id` is a unique value.